libraries/apollo-gradle-plugin-external/src/main/kotlin/com/apollographql/apollo3/gradle/internal/ApolloGenerateSourcesTask.kt

```diff
@@ -213,6 +213,9 @@ abstract class ApolloGenerateSourcesTask : DefaultTask() {
   @get:Input
   abstract val projectName: Property<String>
 
+  @get:Input
+  abstract val projectPath: Property<String>
```

Comment:
Looks like `projectName` isn't used anymore. Can you remove it?

Reply:
It was still used to be passed to Options.moduleName but that was unused too so I removed it :)